packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx

```diff
@@ -34,9 +34,6 @@ export type LabeledInputProps = {
   wrapperProps?: React.ComponentProps<typeof InputGrid>;
   /**
    * Set display style of label.
-   * Supported values:
-   * - 'default' - label appears above input.
-   * - 'inline' - appears in the same line as input.
```

Comment:
Why was this removed here and in `InputGrid.tsx`?

Reply:
This style of comments is not used anywhere else with few options. Removed for consistency.

Reply:
but it's useful information 
<img width="473" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/73ae289a-61f3-4559-9a16-ea499df7911b">

Reply:
also used in other places.

https://github.com/iTwin/iTwinUI/blob/5e46f431232a186968afa9c4b76d28604af64f5f/packages/itwinui-react/src/core/utils/components/Portal.tsx#L29-L33

https://github.com/iTwin/iTwinUI/blob/5e46f431232a186968afa9c4b76d28604af64f5f/packages/itwinui-react/src/core/utils/components/Icon.tsx#L13C2-L17

https://github.com/iTwin/iTwinUI/blob/5e46f431232a186968afa9c4b76d28604af64f5f/packages/itwinui-react/src/core/NotificationMarker/NotificationMarker.tsx#L16-L22

Reply:
Added back. It has some css issues in website.

Reply:
shouldn't the css be fixed then?